### DISCUSSION TITLE
nasıl ekstra puan kazanılır?

### DISCUSSION THREAD

**Discussion by coincexe** [IBAX-io/go-ibax#1357] [Rewards]
arkadaşlar ekstra bir şey bilen var mı puanları nasıl arttırabiliriz?

**Comment by voddah28** [comment]
ref

**Comment by Odysseusksk** [comment]
Amaç airdroptan maksimum sayıda IBXC almaksa, toplanılan puanlar kadar, referans yaratma da önemlidir. Ayrıca GitHub da bir diğer puan kaynağı tabiiki

**Reply by Ozlozlm** [reply]
Gıthub nasıl bağlayacağım puanım gelmedi github a katıldım ama

**Comment by Abdullahoda** [comment]
gıthup da sorunları çözüme kavuşturman gerek.

**Comment by Mehmettkarra** [comment]
Githup dan 3000 geldi başka birşey varmı?

**Comment by Mehmettkarra** [comment]
Arkadaşlar sabırla bekleyelim güzel bir proje yolun başındayken katılmış olmanın illaki bir faydası olacaktır. Borsaya açılacağı zaman ayrıcalık tanısa o bile yeterli
